frontend/src/component/playground/Playground/PlaygroundResultsTable/FeatureStatusCell/FeatureStatusCell.tsx

```diff
@@ -10,7 +10,7 @@ interface IFeatureStatusCellProps {
 const StyledCellBox = styled(Box)(({ theme }) => ({
     display: 'flex',
     alignItems: 'center',
-    padding: theme.spacing(1, 2),
+    padding: theme.spacing(1, 0, 1, 2),
```

Comment:
spacing between status chip and info was too big. I tested and this change doesn't affect the old playground.